lib/scrape.js

```diff
@@ -276,15 +370,26 @@ function checkGitHubUserCacheExpired(user) {
 }
 
 async function freshenUserGitHubCache(user, existingUser, organization) {
-  if (!(existingUser && existingUser.github_updated)) {
+  if (
+    !(
+      existingUser &&
+      existingUser.github_updated &&
```

Comment:
See <https://github.com/airbnb/javascript#control-statements>.

Reply:
I agree with Airbnb, but this is a styling thing in our ESLint config that we should probably change in another PR.